api/routes/index.js

```diff
@@ -1,4 +1,27 @@
-import express from 'express';
-const router = express.Router();
+export function register(server, options, next) {
+  /* Serve static files from public folder */
+  server.route({
```

Comment:
I'm happy to move the public endpoint out. Ideally I was thinking the register function in `api/index` would pull in each route file.